### ISSUE TITLE
inserting gas tank in canister doesn't make a sound

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#32835]
https://github.com/user-attachments/assets/db97d289-895a-49f3-87c1-b83074f1f4ff

**Comment by scrivoy** [comment]
Good catch! I might have a possible solution for that in my head. I'll check on it tonight or the next days.

**Comment by scrivoy** [comment]
> Good catch! I might have a possible solution for that in my head. I'll check on it tonight or the next days.

Okay, I was a bit too optimistic. I think this might be a bit too difficult for me.
But I'll still try to mess around with it, although I don't think I can fix it.
I need to learn more C#. :)

**Comment by Alpaccalypse** [comment]
Ejecting with valve open on a full canister should ideally have a loud and disturbing sound indication too. I should probably make it as a separate issue, as it will be more complicated that the insert sound, as ideally it should vary according to pressure and check the canister isn't empty.

**Comment by Alpaccalypse** [comment]
> > Good catch! I might have a possible solution for that in my head. I'll check on it tonight or the next days.
> 
> Okay, I was a bit too optimistic. I think this might be a bit too difficult for me. But I'll still try to mess around with it, although I don't think I can fix it. I need to learn more C#. :)

There is an `enum.GasCanisterVisuals.TankInserted` which is used for the sprite layers. I'm not sure if it can only be used for visuals, but if not, this might work with a sound trigger, which would mean you could sidestep the C# part.
